geonode/qgis_server/tests/test_helpers.py

```diff
@@ -251,22 +252,21 @@ def test_delete_orphan(self):
         # register file list
         layer_path = settings.QGIS_SERVER_CONFIG['layer_directory']
         tiles_path = settings.QGIS_SERVER_CONFIG['tiles_directory']
-        geonode_layer_path = os.path.join(settings.MEDIA_ROOT, 'layers')
 
-        qgis_layer_list = set(os.listdir(layer_path))
-        tile_cache_list = set(os.listdir(tiles_path))
-        geonode_layer_list = set(os.listdir(geonode_layer_path))
+        qgis_layers = set(os.listdir(layer_path))
```

Comment:
I wouldn't usually go as far as renaming existing variables in a large project like this just for the sake of semantics. 

Keep it for now given that this is in a test file and not used anywhere, but I think the improvements are rather minimal :)